test/fuzz/env.ml

```diff
@@ -8,11 +8,13 @@ type t =
   ; mutable next_fun : int
   ; mutable next_local : int
   ; mutable next_block : int
+  ; mutable next_loop : int
   ; mutable datas : string list
   ; mutable memory : string option
   ; mutable globals : (string * global_type) list
   ; mutable locals : (string * val_type) list
   ; mutable blocks : (string * block_type) list
+  ; mutable loops : (string * block_type) list
```

Comment:
Why do you separate `blocks` and `loops` ?

Reply:
We should have something like: `mutable blocks : (block_kind, string, block_type) list` and only this (not `loops` and `blocks`).